### ISSUE TITLE
PlatoPythonModule build error: identifier "Py_InitModule3" is undefined. CMakeLists find_package should specify which version of Python is required.

### ISSUE THREAD

**Issue by p-ranav** [SNLComputation/lgrtk#25]
From the looks of it, the PlatoPythonModule build requires Python 2.7. This needs to be specified in the CMakeLists to prevent cmake from finding and using Python 3.x on machines that have multiple versions of Python installed. 

```
option(LGR_ENABLE_PLATO_PYTHON "Enable the Plato Python interface" OFF)
if (LGR_ENABLE_PLATO_PYTHON)
    find_package(PythonLibs)
    include_directories( ${PYTHON_INCLUDE_DIRS} )
    add_library( Plato SHARED src/plato/LGR_App.cpp src/plato/PlatoPythonModule.cpp )
    target_include_directories( Plato PRIVATE "${CMAKE_CURRENT_SOURCE_DIR}/src")
    target_include_directories( Plato PRIVATE "${PLATO_PREFIX}/include")
    target_link_libraries( Plato lgrtk ${PLATO_LIBS} ${Trilinos_LIBRARIES} ${Trilinos_TPL_LIBRARIES} ${PYTHON_LIBRARIES} )
    set_target_properties( Plato PROPERTIES PREFIX "" )
    target_compile_options( Plato PRIVATE "-lmpi" )
    if( CMAKE_INSTALL_PREFIX )
    install( TARGETS Plato DESTINATION ${CMAKE_INSTALL_PREFIX}/lib )
    endif()
endif()
option(LGR_ENABLE_PLATO_GEOMETRY "Enable Plato Geometry" OFF)
if (LGR_ENABLE_PLATO_GEOMETRY)
add_definitions(-DPLATO_GEOMETRY)
endif()
```

Here's the error I see on spack. Py_InitModule3 does not exist in Python3. 

```
==> Installing platoanalyze
==> Cloning git repository: https://github.com/SNLComputation/lgrtk.git on branch master
==> No checksum needed when fetching with git
==> Already staged platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku in /opt/pranav/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku
==> No patches needed for platoanalyze
==> Building platoanalyze [CMakePackage]
==> Executing phase: 'cmake'
==> Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j20'

4 errors found in build log:
     398    [ 77%] Built target lgr
     399    [ 78%] Building CXX object unit_tests/CMakeFiles/PlatoMathHelpersUnitTests.dir/Teuchos_StandardUnitTestMain.cpp.o
     400    cd /tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build/unit_tests && /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/nvcc-wrapper-master-kz3zjguqndqwzupvqeyhknn3s6vi2gdu/etc
            /nvcc_wrapper  -DHAVE_AMGX -I/usr/include/python3.5m -I/opt/pranav/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src -I/tmp/pranav/spack-stage/spack-stage-nUZGpT/
            lgrtk/spack-build/src -I/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/amgx-master-lltsrhbja2hnzmim2pm4kgwfmbzxr5rl/include -I/opt/pranav/spack/var/spack/stage/platoanalyze-master-zz
            hmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src/solverInterfaces/AmgX -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/omega-h-master-omm6bln6gxou6qaj43ngmyo77ldb5j7f/include -isyste
            m /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/include -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/netc
            df-4.6.1-ase5xze6hcd4h53lj4ydgr53fkhli2fw/include -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/hdf5-1.10.2-h2ecpr3psnei6q4b7suge7qjgxglbgfs/include -isystem /opt/pranav/sp
            ack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/parallel-netcdf-1.8.0-usb4gz3z2htuetb22bm2ma2md3asmgsz/include -isystem /usr/local/cuda/include -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.0
            4-x86_64/gcc-5.4.0/zlib-1.2.11-5nus6knzumx4ik2yl44jxtgtsl7d54xb/include  -expt-extended-lambda -std=c++11 -std=c++11   -O2 -g -DNDEBUG   -o CMakeFiles/PlatoMathHelpersUnitTests.dir/Teuchos_StandardU
            nitTestMain.cpp.o -c /opt/pranav/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/unit_tests/Teuchos_StandardUnitTestMain.cpp
     401    nvcc_wrapper - *warning* you have set multiple optimization flags (-std=c++1* or --std=c++1*), only the first is used because nvcc can only accept a single std setting
     402    /opt/pranav/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src/plato/PlatoPythonModule.cpp(396): warning: non-void function "initPlato" should return a value
     403    
  >> 404    /opt/pranav/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src/plato/PlatoPythonModule.cpp(398): error: identifier "Py_InitModule3" is undefined
     405    
     406    /opt/pranav/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src/plato/PlatoPythonModule.cpp(403): warning: missing return statement at end of non-void function "init
            Plato"
     407    
     408    1 error detected in the compilation of "/tmp/tmpxft_000057f1_00000000-6_PlatoPythonModule.cpp1.ii".
     409    CMakeFiles/Plato.dir/build.make:78: recipe for target 'CMakeFiles/Plato.dir/src/plato/PlatoPythonModule.cpp.o' failed
  >> 410    make[2]: *** [CMakeFiles/Plato.dir/src/plato/PlatoPythonModule.cpp.o] Error 1
     411    make[2]: *** Waiting for unfinished jobs....
     412    [ 79%] Building CXX object CMakeFiles/LGR_MPMD.dir/src/plato/LGR_MPMD.cpp.o
     413    /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/nvcc-wrapper-master-kz3zjguqndqwzupvqeyhknn3s6vi2gdu/etc/nvcc_wrapper  -DHAVE_AMGX -I/opt/pranav/spack/var/spack/stage/platoanalyze-mas
            ter-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src -I/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/platoengine-master-z6alq3hugu7pdhkkvjewpk6zvabm3t7a/include -I/usr/include/python3.5m 
            -I/tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build/src -I/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/amgx-master-lltsrhbja2hnzmim2pm4kgwfmbzxr5rl/include -I/opt/prana
            v/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src/solverInterfaces/AmgX -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/omega-h-master-om
            m6bln6gxou6qaj43ngmyo77ldb5j7f/include -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/include -isystem /opt/pranav/spack/opt
            /spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/netcdf-4.6.1-ase5xze6hcd4h53lj4ydgr53fkhli2fw/include -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/hdf5-1.10.2-h2ecpr3psnei6q4b7s
            uge7qjgxglbgfs/include -isystem /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/parallel-netcdf-1.8.0-usb4gz3z2htuetb22bm2ma2md3asmgsz/include -isystem /usr/local/cuda/include -isyste
            m /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/zlib-1.2.11-5nus6knzumx4ik2yl44jxtgtsl7d54xb/include  -expt-extended-lambda -std=c++11 -std=c++11   -O2 -g -DNDEBUG   -o CMakeFiles/L
            GR_MPMD.dir/src/plato/LGR_MPMD.cpp.o -c /opt/pranav/spack/var/spack/stage/platoanalyze-master-zzhmhtoyd6iezsu4xfjplnqnyeqbleku/lgrtk/src/plato/LGR_MPMD.cpp
     414    nvcc_wrapper - *warning* you have set multiple optimization flags (-std=c++1* or --std=c++1*), only the first is used because nvcc can only accept a single std setting
     415    [ 80%] Linking CXX executable ROCKET_MPMD
     416    /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/cmake-3.12.1-lk3pxtstogd2pz6ueafzkbnzmhooi6uu/bin/cmake -E cmake_link_script CMakeFiles/ROCKET_MPMD.dir/link.txt --verbose=1

     ...

     503    [ 98%] Linking CXX executable LGRAppIntxTests
     504    cd /tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build/unit_tests && /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/cmake-3.12.1-lk3pxtstogd2pz6ueafzkbnzmhooi6uu/bin/cmake 
            -E cmake_link_script CMakeFiles/LGRAppIntxTests.dir/link.txt --verbose=1
     505    /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/nvcc-wrapper-master-kz3zjguqndqwzupvqeyhknn3s6vi2gdu/etc/nvcc_wrapper    -expt-extended-lambda -std=c++11 -std=c++11   -O2 -g -DNDEBUG 
             -rdynamic CMakeFiles/LGRAppIntxTests.dir/__/src/plato/LGR_App.cpp.o CMakeFiles/LGRAppIntxTests.dir/LGRAppIntxTests.cpp.o CMakeFiles/LGRAppIntxTests.dir/LGRTestHelpers.cpp.o CMakeFiles/LGRAppIntxTes
            ts.dir/Teuchos_LGRAppUnitTestMain.cpp.o  -o LGRAppIntxTests -Wl,-rpath,/tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build/src:/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.
            0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib:/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/matio-1.5.9-tnu7atbvn35ttzfvlqvxrlwfm3t5br5n/lib:/opt/pranav/spack/opt/spack/lin
            ux-ubuntu16.04-x86_64/gcc-5.4.0/netcdf-4.6.1-ase5xze6hcd4h53lj4ydgr53fkhli2fw/lib:/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/hdf5-1.10.2-h2ecpr3psnei6q4b7suge7qjgxglbgfs/lib:/opt
            /pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/zlib-1.2.11-5nus6knzumx4ik2yl44jxtgtsl7d54xb/lib:/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/openblas-0.3.2-cmdvstekzc4e
            uqcteiwdog7p3doivigo/lib:/usr/local/cuda/lib64:/opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/omega-h-master-omm6bln6gxou6qaj43ngmyo77ldb5j7f/lib:/opt/pranav/spack/opt/spack/linux-ub
            untu16.04-x86_64/gcc-5.4.0/amgx-master-lltsrhbja2hnzmim2pm4kgwfmbzxr5rl/lib ../src/liblgrtk.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/platoengine-master-z6alq3hugu7pdhkkvjewp
            k6zvabm3t7a/lib/libPlatoDataLayer.a /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/platoengine-master-z6alq3hugu7pdhkkvjewpk6zvabm3t7a/lib/libPlatoInterface.a /opt/pranav/spack/opt/s
            pack/linux-ubuntu16.04-x86_64/gcc-5.4.0/platoengine-master-z6alq3hugu7pdhkkvjewpk6zvabm3t7a/lib/libPlatoAnalyze.a /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/platoengine-master-z6
            alq3hugu7pdhkkvjewpk6zvabm3t7a/lib/libPlatoParser.a /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/platoengine-master-z6alq3hugu7pdhkkvjewpk6zvabm3t7a/lib/libPlatoExceptions.a /opt/p
            ranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libintrepid.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/tril
            inos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libModeLaplace.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libanasaz
            iepetra.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libanasazi.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04
            -x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libmapvarlib.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pf
            obw6dr/lib/libsuplib_cpp.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libsuplib_c.so.12.13 /opt/pranav/spack/opt/spack
            /linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libsuplib.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7
            hsqnefkcfe2z3pfobw6dr/lib/libsupes.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libaprepro_lib.so.12.13 /opt/pranav/sp
            ack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libchaco.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master
            -r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libio_info_lib.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIonit.so.12.13 /o
            pt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIotr.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/tril
            inos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIohb.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIogs.so.12.13
             /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIogn.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/t
            rilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIovs.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIoexo_fac.
            so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIopx.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc
            -5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIofx.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIo
            ex.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libIoss.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/
            gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libnemesis.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib
            /libexoIIv2for32.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libexodus_for.so.12.13 /opt/pranav/spack/opt/spack/linux
            -ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libbelosepetra.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7h
            sqnefkcfe2z3pfobw6dr/lib/libbelos.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libml.so.12.13 /opt/pranav/spack/opt/sp
            ack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libaztecoo.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3
            naz7hsqnefkcfe2z3pfobw6dr/lib/libepetraext.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libtriutils.so.12.13 /opt/pran
            av/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libshards.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-
            master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libepetra.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libsacado.so.12.13 
            /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libteuchoskokkoscomm.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64
            /gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libteuchoskokkoscompat.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z
            3pfobw6dr/lib/libteuchosremainder.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libteuchosnumerics.so.12.13 /opt/pranav
            /spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libkokkosalgorithms.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/t
            rilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libkokkoscontainers.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/matio-1.5.9-tnu7atbvn35ttzfvlqvxrlwfm3t5br5n/lib/libma
            tio.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/netcdf-4.6.1-ase5xze6hcd4h53lj4ydgr53fkhli2fw/lib/libnetcdf.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/pa
            rallel-netcdf-1.8.0-usb4gz3z2htuetb22bm2ma2md3asmgsz/lib/libpnetcdf.a /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/hdf5-1.10.2-h2ecpr3psnei6q4b7suge7qjgxglbgfs/lib/libhdf5.so /opt/
            pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/zlib-1.2.11-5nus6knzumx4ik2yl44jxtgtsl7d54xb/lib/libz.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/hdf5-1.10.2-h2ecpr3p
            snei6q4b7suge7qjgxglbgfs/lib/libhdf5_hl.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/openblas-0.3.2-cmdvstekzc4euqcteiwdog7p3doivigo/lib/libopenblas.so -ldl /usr/local/cuda/lib6
            4/libcudart.so /usr/local/cuda/lib64/libcublas.so /usr/local/cuda/lib64/libcufft.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/omega-h-master-omm6bln6gxou6qaj43ngmyo77ldb5j7f/lib
            /libomega_h.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/zlib-1.2.11-5nus6knzumx4ik2yl44jxtgtsl7d54xb/lib/libz.so -L/usr/lib/gcc/x86_64-linux-gnu/7/ -lgfortran /opt/pranav/spack
            /opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/amgx-master-lltsrhbja2hnzmim2pm4kgwfmbzxr5rl/lib/libamgxsh.so -lm /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3
            naz7hsqnefkcfe2z3pfobw6dr/lib/libexodus.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/netcdf-4.6.1-ase5xze6hcd4h53lj4ydgr53fkhli2fw/lib/libnetcdf.so /opt/pranav/spack/opt/s
            pack/linux-ubuntu16.04-x86_64/gcc-5.4.0/parallel-netcdf-1.8.0-usb4gz3z2htuetb22bm2ma2md3asmgsz/lib/libpnetcdf.a /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3
            naz7hsqnefkcfe2z3pfobw6dr/lib/libteuchoscomm.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libteuchosparameterlist.so.1
            2.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libteuchosparser.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_6
            4/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6dr/lib/libteuchoscore.so.12.13 /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/trilinos-master-r3pkbz3naz7hsqnefkcfe2z3pfobw6
            dr/lib/libkokkoscore.so.12.13 -ldl /usr/local/cuda/lib64/libcudart.so /usr/local/cuda/lib64/libcublas.so /usr/local/cuda/lib64/libcufft.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.
            4.0/hdf5-1.10.2-h2ecpr3psnei6q4b7suge7qjgxglbgfs/lib/libhdf5.so /opt/pranav/spack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/zlib-1.2.11-5nus6knzumx4ik2yl44jxtgtsl7d54xb/lib/libz.so /opt/pranav/sp
            ack/opt/spack/linux-ubuntu16.04-x86_64/gcc-5.4.0/hdf5-1.10.2-h2ecpr3psnei6q4b7suge7qjgxglbgfs/lib/libhdf5_hl.so
     506    nvcc_wrapper - *warning* you have set multiple optimization flags (-std=c++1* or --std=c++1*), only the first is used because nvcc can only accept a single std setting
     507    make[2]: Leaving directory '/tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build'
     508    CMakeFiles/Makefile2:368: recipe for target 'CMakeFiles/Plato.dir/all' failed
  >> 509    make[1]: *** [CMakeFiles/Plato.dir/all] Error 2
     510    make[1]: *** Waiting for unfinished jobs....
     511    make[2]: Leaving directory '/tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build'
     512    [ 98%] Built target LGR_MPMD
     513    make[2]: Leaving directory '/tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build'
     514    [ 98%] Built target LGRAppIntxTests
     515    make[1]: Leaving directory '/tmp/pranav/spack-stage/spack-stage-nUZGpT/lgrtk/spack-build'
     516    Makefile:143: recipe for target 'all' failed
  >> 517    make: *** [all] Error 2
```

**Comment by ibaned** [comment]
@maguilo11 @jrobbin

**Comment by jrobbin** [comment]
This issue was fixed in commit 75e0e7f7 on 2019-04-03.
